src/sql/src/kafka_util.rs

```diff
@@ -235,16 +229,11 @@ pub async fn create_consumer(
     // ensure that at least one metadata request succeeds.
     task::spawn_blocking(move || format!("kafka_get_metadata:{topic}"), {
         let consumer = Arc::clone(&consumer);
-        move || {
-            let _ = consumer.fetch_metadata(Some(&owned_topic), Duration::from_secs(2));
-        }
+        move || consumer.fetch_metadata(Some(&owned_topic), Duration::from_secs(2))
```

Comment:
Think we should bump this timeout to be more than 2s. What do we do elsewhere? 30s? We should factor that timeout into a `kafka_util` constant so that it's shared everywhere.

Reply:
I believe we retry this at a higher-level, let me check...

Reply:
Ah, the `create_with_context` is retried, yes Ill update this to 30s!

Reply:
Updated with the same timeout in adapter/sources/sinks